postgres/src/main/java/com/google/cloud/sql/postgres/SocketFactory.java

```diff
@@ -47,7 +53,15 @@ public SocketFactory(String instanceName) {
   @Override
   public Socket createSocket() throws IOException {
     logger.info(String.format("Connecting to Cloud SQL instance [%s].", instanceName));
-    return SslSocketFactory.getInstance().create(instanceName);
+    String runtime = System.getenv(GaeRuntime);
+
+    if (runtime == null || runtime.isEmpty()) {  // we aren't on Standard (not set local)
```

Comment:
What do you mean by ` (not set local)`? I understand that you are saying that the environmental variable with be set if and only if we are on standard, but I am confused by the parenthetical after that.